### PR TITLE
[Feature] Generate Season Statistics for Players

### PR DESCRIPTION
**Pull request by rzwoebkenberg**
### Deliverable
- [stats.csv](https://github.com/SHiZ-SHiZ/rocket_league_stats/files/9439016/stats.csv) containing player statistics for a season (10 games)

### Method
- RLCSV BakkesMod plugin used to generate a csv with individual statistics for each game played. Script iterates through each file, and makes use of string splits to create a list unique to each player's performance in a match. A struct for each player is the latter of a key-value pair dictionary, where the various statistics are kept. Iterate through the dictionary by key to access each player object and interact with the class variables 

### Testing
- ask Ryan for game files from testing season. 
- Follow documentation instructions in the included README.md
- Verify correct totals
- Indicate deviations from expected results and gaps in the User's Guide

### TODO
- ~~make paths relative. Maybe we keep the game files in the repo too?~~
     - **relative paths added**
- script modification to support game by game results for the other part of the sheet.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,42 @@
...
+
+&emsp;4.) Play the games you want to record. At the conclusion of each game, the results will be saved as a .csv in the directory you specified in step 2. When you open the directory, it should look like the below figure.
+
+<p align="center">
+  <img src="documentation/pics/directory.png" />
+</p>
+
+&emsp;5.) Download or clone the code from the repository. \
+&emsp;6.) Navigate to the tools folder inside the repository. Shift + Right-Click in the empty space inside the folder and click Open PowerShell window here. \
+&emsp;7.) Run the following command, replacing ```"INSERT_PATH_HERE"``` with the exact path entered during step 2:
```
Comment (JD0517) [inline README.md:32]: Should be "during step 3:"

File: README.md [hunk 2]
```diff
@@ -1 +1,42 @@
...
+
+```
+python season-stats.py "INSERT_PATH_HERE"
+```
+
+&emsp;&emsp;If that doesn't work, try replacing ```py``` with ```python```
```
Comment (JD0517) [inline README.md:38]: py and python should be swapped here

**Comment by JD0517** [comment]
There is no clear indication as to what each column represents. This should either be added as a header row in the stats.csv or explained in the documentation.

**Comment by JD0517** [comment]
Maybe move the rl stats plugin out of the documentation folder

**Comment by JD0517** [comment]
> 

This also applies to which team each player is on.

**Comment by rzwoebkenberg** [comment]
> There is no clear indication as to what each column represents. This should either be added as a header row in the stats.csv or explained in the documentation.

Column numbers are in the top of the script as comments for now. The user will not need to interact with the generated `stats.csv` file

**Review by JD0517 — APPROVED** [review]
Nice!

**Pull request merged**